orchestrator/src/main/java/com/scottlogic/datahelix/generator/orchestrator/generate/GenerateCommandLine.java

```diff
@@ -46,9 +47,9 @@
     descriptionHeading = "%nDescription:%n",
     parameterListHeading = "%nParameters:%n",
     optionListHeading = "%nOptions:%n",
+    version = { "Profile Schema Version " + ProfileConfiguration.PROFILE_SCHEMA_VERSION},
```

Comment:
We have two versions, the schema version and the datahelix version. How should this be displayed?

Reply:
As discussed at standup, we don't at present have any way to access the DH version in code, as it's only set in GitHub.